components/Navbar.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import Link from "next/link";
+import TodoIcon from "@/public/todoIcon.svg";
+import ProgressionIcon from "@/public/progressIcon.svg";
+import DoneIcon from "@/public/doneIcon.svg";
+export default function Navbar() {
+  return (
+    <StyledNavbar>
```

Comment:
Would be nice to have some kind of indicator to show which Page I am currently on. I assume you are going to implement that later :)

Reply:
Yes I just added some basic styling for now.